crypto/ring-signature/src/amount/commitment.rs

```diff
@@ -3,14 +3,21 @@
 use crate::{CompressedCommitment, Error, PedersenGens, Scalar};
 use core::fmt;
 use curve25519_dalek::ristretto::{CompressedRistretto, RistrettoPoint};
+
+#[cfg(feature = "serde")]
+use serde::{Deserialize, Serialize};
+
```

Comment:
Nit: Remove blank lines, sort
```suggestion
#[cfg(feature = "serde")]
use serde::{Deserialize, Serialize};
```